source/_components/sensor.bme680.markdown

```diff
@@ -111,6 +111,11 @@ aq_humidity_bias:
   required: false
   default: 25
   type: integer
+temp_offset:
+  description: The temperature for the sensor will always be too high as it pulls heat from the components around it.  Consider adding a negative offset to ensure the sensor returns an accurate temperature. Note: This value is in celsius.
```

Comment:
This will solve the build error 😉 
```suggestion
  description: "The temperature for the sensor will always be too high as it pulls heat from the components around it.  Consider adding a negative offset to ensure the sensor returns an accurate temperature. Note: This value is in celsius."
```